package.json

```diff
@@ -8,7 +8,11 @@
     "lint:js": "eslint --ext .js,.vue,.html calendar datepicker tests",
     "lint:sass": "sass-lint -q -v",
     "pretest": "npm run lint",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "docs": "nuxt -c docs/nuxt.config.js",
+    "docs:generate": "nuxt generate -c docs/nuxt.config.js",
+    "docs:gh-pages": "gh-pages --dotfiles --dist docs/dist",
```

Comment:
I've added a `.travis.yml` file that will deploy to GitHub Pages from the `dist` folder on successful merge in to `master`. Let's use that instead of the `gh-pages` module?

Reply:
Hmm let me check

Reply:
I've rebased and made few changes to generate docs on `script` phase. Are they ok?